packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -27,16 +27,19 @@ import { IDetailedHbarSpendingPlan } from '../../db/types/hbarLimiter/hbarSpendi
 import { HbarSpendingPlanRepository } from '../../db/repositories/hbarLimiter/hbarSpendingPlanRepository';
 import { EthAddressHbarSpendingPlanRepository } from '../../db/repositories/hbarLimiter/ethAddressHbarSpendingPlanRepository';
 import { IPAddressHbarSpendingPlanRepository } from '../../db/repositories/hbarLimiter/ipAddressHbarSpendingPlanRepository';
+import constants from '../../constants';
+import { Hbar } from '@hashgraph/sdk';
 
 export class HbarLimitService implements IHbarLimitService {
-  static readonly ONE_DAY_IN_MILLIS = 24 * 60 * 60 * 1000;
-  // TODO: Replace with actual values
-  static readonly DAILY_LIMITS: Record<SubscriptionType, number> = {
-    BASIC: parseInt(process.env.HBAR_DAILY_LIMIT_BASIC ?? '1000'),
-    EXTENDED: parseInt(process.env.HBAR_DAILY_LIMIT_EXTENDED ?? '10000'),
-    PRIVILEGED: parseInt(process.env.HBAR_DAILY_LIMIT_PRIVILEGED ?? '100000'),
+  // TODO: Replace with actual values - https://github.com/hashgraph/hedera-json-rpc-relay/issues/2895
+  static readonly DAILY_LIMITS: Record<SubscriptionType, Hbar> = {
```

Comment:
Q: are they references to daily limits accurate or are they variable based on the duration?

Reply:
Good point - this variable name is no longer correct, I can rename it to `TIERED_LIMITS`.